src/libstd/keyword_docs.rs

```diff
@@ -558,6 +558,32 @@ mod impl_keyword { }
 /// [Reference]: ../reference/statements.html#let-statements
 mod let_keyword { }
 
+#[doc(keyword = "while")]
+//
+/// while loop
+///
+/// `while` is used to define while loops in Rust. It runs the code inside it
+/// until the condition in the while loop becomes False or there the code uses
+/// `break`.
+///
+/// ```rust
+/// while true {
```

Comment:
Suggest using `loop` here instead in a comment?  
Maybe move this example to the bottom, as it's uncommon due to `loop` being available.

Reply:
You might need to set the `no_run` flag for this code block as well, as it would infinitely block on doc tests otherwise.

Reply:
It started to look as a bad example to me, so I removed it.